eng/common/scripts/Create-APIReview.ps1

```diff
@@ -116,14 +116,15 @@ foreach ($pkgName in $responses.Keys)
                 {
                     Write-Host "Package version $($version) is GA and automatic API Review is not yet approved for package $($PackageName)."
                     Write-Host "Build and release is not allowed for GA package without API review approval."
-                    Write-Host "Please reach out to API review approvers and share above link to review for approval."
+                    Write-Host "Please send email to adparchrescue@microsoft.com and share above link to API review for emergency approval to release you package."
```

Comment:
Instead of directly putting an email address here it is probably just better to refer to the faq.